nixos/modules/module-list.nix

```diff
@@ -439,6 +439,7 @@
   ./services/audio/spotifyd.nix
   ./services/audio/squeezelite.nix
   ./services/audio/tts.nix
+  ./services/audio/upmpdcli.nix
```

Comment:
Could you add a release note to `nixos/doc/manual/release-notes/rl-2511.section.md`?